data/transactions/logic/eval.go

```diff
@@ -585,6 +588,36 @@ func opMul(cx *evalContext) {
 	cx.stack = cx.stack[:last]
 }
 
+func opMulwImpl(x, y uint64) (uint64, uint64, error) {
```

Comment:
it would be nice to name the return fields (high, low, err)

Reply:
done